core/include/detray/navigation/direct_navigator.hpp

```diff
@@ -222,12 +222,17 @@ class direct_navigator {
         }
 
         DETRAY_HOST_DEVICE
-        inline auto abort() -> bool {
+        inline auto abort(const char * = nullptr) -> bool {
```

Comment:
Is there a reason this doesn't use the string at all?

Reply:
The direct navigator does not have an inspector, but it still needs a matching call interface to compile